hack/verify-govet.sh

```diff
@@ -23,13 +23,69 @@ FAILURE=false
 test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
 for test_dir in $test_dirs
 do
-  go tool vet -shadow=false $test_dir
+  go tool vet -all $test_dir
```

Comment:
no. the flags to `go vet` changed with the latest release.
`-all` == all, *except* `shadow`